bigio/utils.js

```diff
@@ -100,42 +100,42 @@ module.exports = {
         return '';
     },
 
-    getIp: function(cb) {
-        if(ip === undefined) {
-            if(config.nic === undefined) {
-                var interfaces = os.networkInterfaces();
-                var match;
-
-                switch(currentOS()) {
-                    case OperatingSystem.WIN_64:
-                    case OperatingSystem.WIN_32:
-                        match = "Loopback";
-                        break;
-                    case OperatingSystem.LINUX_64:
-                    case OperatingSystem.LINUX_32:
-                        match = "lo";
-                        break;
-                    case OperatingSystem.MAC_64:
-                    case OperatingSystem.MAC_32:
-                        match = "lo0";
-                        break;
-                    default:
-                        logger.error("Cannot determine operating system. Cluster cannot form.");
-                }
-
-                for(var intfc in interfaces) {
-                    if(intfc.indexOf(match) > -1) {
-                        for(var i in interfaces[intfc]) {
-                            if(interfaces[intfc][i].family == 'IPv4') {
-                                ip = interfaces[intfc][i].address;
-                            }
-                        }
-                    }
-                }
+    getIp: function (cb) {
```

Comment:
The indent of this function is only two spaces, while the rest of the file is four. Please either increase to four, or decrease the rest of the file to two spaces to maintain consistency.